test/fixtures/amp-cupcake-ad.html

```diff
@@ -1,38 +1,1014 @@
-</style></head><body marginwidth="0" marginheight="0" class="amp-mode-mouse" style="opacity: 1; visibility: visible; animation: none 0s ease 0s 1 normal none running;"><amp-pixel allow-ssr-img="" class="i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element" i-amphtml-layout="fixed" i-amphtml-ssr="" src="http:__VIEW_URL__" style="width:1px;height:1px;" aria-hidden="true"><img src="http:__VIEW_URL__"></amp-pixel><a href="https://google.com/" target="_blank">
-    <amp-img class="i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element i-amphtml-layout" height="200" i-amphtml-layout="fixed" i-amphtml-ssr="" src="http://localhost:__TEST_SERVER_PORT__/test/fixtures/served/cupcake.jpg" style="width:300px;height:200px;" width="300">
-    <img class="i-amphtml-fill-content i-amphtml-replaced-content" decoding="async" src="http://localhost:__TEST_SERVER_PORT__/test/fixtures/served/cupcake.jpg"></amp-img>
-  </a>
-  <amp-analytics class="i-amphtml-layout-fixed i-amphtml-layout-size-defined i-amphtml-element" i-amphtml-layout="fixed" style="width:1px;height:1px;" trigger="immediate" aria-hidden="true" hidden="">
-    <script type="application/json">
-      {
-        "transport": {"beacon": false, "xhrpost": false},
-        "requests": {
-          "visibility": "http:__VISIBLE_URL__"
-        },
-        "triggers": {
-          "pageVisible": {
-            "on": "visible",
-            "request": "visibility",
-            "vars": { "type": "pageVisible" }
+<html
```

Comment:
Is this file changed due to running prettier, or are there other differences?

Reply:
@caroqliu purely prettier